orderer/common/multichannel/follower_resources.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright IBM Corp. 2017 All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package multichannel
+
+import (
+	"github.com/hyperledger/fabric-protos-go/common"
+	"github.com/hyperledger/fabric/bccsp"
+	"github.com/hyperledger/fabric/common/channelconfig"
+	"github.com/hyperledger/fabric/common/ledger/blockledger"
+	"github.com/hyperledger/fabric/common/policies"
+	"github.com/hyperledger/fabric/internal/pkg/identity"
+	"github.com/hyperledger/fabric/orderer/consensus"
+	"github.com/hyperledger/fabric/protoutil"
+	"github.com/pkg/errors"
+)
+
+// FollowerResources encapsulates all the resources the follower needs to do its job:
+// ledger & config, membership detector, signer & block-verifier.
+type FollowerResources struct {
+	*ledgerResources                              // ledger and config resources
+	identity.SignerSerializer                     // sign requests to the delivery service of other orderers
+	BCCSP                     bccsp.BCCSP         // verify messages
+	consenter                 consensus.Consenter // detect whether a block indicates channel membership
+}
+
+func NewFollowerResources(
+	ledgerResources *ledgerResources,
+	consenter consensus.Consenter,
+	signer identity.SignerSerializer,
+	bccsp bccsp.BCCSP,
+) *FollowerResources {
+	fRes := &FollowerResources{
+		ledgerResources:  ledgerResources,
+		SignerSerializer: signer,
+		BCCSP:            bccsp,
+		consenter:        consenter,
+	}
+
+	return fRes
+}
+
+// TODO Move this method to ledgerResources struct, so it can be shared with the ChainSupport struct.
```

Comment:
The ledgerResources might be the right place to hang this, but, please don't be afraid to break up and or refactor the 'multichannel.configReources' type and its children, that structure was poorly thought, hard to follow, and now might be the time to refactor it.

Reply:
I agree. I would like to untangle the `ledgerResources` step by step, in a PR of its own, for safety, as it affects all the other chains.

Reply:
Sounds good to me.